src/psyclone/psyir/nodes/schedule.py

```diff
@@ -67,12 +75,11 @@ def _validate_child(position, child):
         # pylint: disable=unused-argument
         return isinstance(child, Statement)
 
-    def __init__(self, children=None, parent=None):
+    def __init__(self, children=None, parent=None, symbol_table=None):
```

Comment:
Can `children` and `parent` now be removed as optional arguments?

Reply:
They can be, but I would rather not, I think parent and children are a common interface to almost all nodes. If any is removed should be part of a different discussion and applied to every node.

Reply:
This is (partially?) the subject of #645.

Reply:
Yes. I think #645 emerged from my request and Sergi not wanting to do it. I accept the argument and drop the request.